src/app/lib/config/services/gahuza.js

```diff
@@ -142,31 +143,36 @@ const service = {
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: "Soma ibijanye n'aho duhagaze ku mihora ijana ahandi",
       },
       links: [
         {
           href: 'https://www.bbc.com/news/help-41670342',
```

Comment:
This one has changed too.
```suggestion
          href: 'https://www.bbc.com/gahuza/institutional-49283343',
```